sources/platform/api_v2/api_v2_reference.apib

```diff
@@ -3489,6 +3489,8 @@ To assign the new webhook to an actor or task, the request body must contain `re
 * `eventTypes` is a list of events that will trigger the webhook, e.g. when the actor run succeeds.
 * `condition` should be an object containing the ID of the actor or task to which the webhook will be assigned.
 * `payloadTemplate` is a JSON-like string, whose syntax is extended with the use of variables.
+* `headersTemplate` is a JSON-like string, whose syntax is extended with the use of variables. Following values will be re-written to defaults: "host", "Content-Type", "X-Apify-Webhook", "X-Apify-Webhook-Dispatch-Id", "X-Apify-Request-Origin"
```

Comment:
I have a bit of pedantic beef with the phrase `JSON-like string`, since really it's stringified JSON. Could we change this for `stringified JSON`?

Reply:
It does not have to be stringified json, JSON implies encapsulating values in quotes, which you do not actually have to do.

Reply:
ahh, well in that case, the beef is squashed :D